src/pages/profile.tsx

```diff
@@ -81,7 +82,14 @@ const ProfilePage: NextPage<Props> = ({ chaptersData }) => {
           <div className={styles.container}>
             <div className={classNames(layoutStyle.flowItem)}>
               <div className={styles.profileContainer}>
-                <div className={styles.profilePicture} />
+                <div className={styles.profilePicture}>
+                  {/* eslint-disable-next-line @next/next/no-img-element */}
+                  <img
```

Comment:
qq: why aren't we using next's Image component?

Reply:
to use Next image we need to configure the domain.

otherwise we'll get 
```
Invalid src prop [(https://www.gravatar.com/avatar/00000000000000000000000000000000?d=mp&f=y)](https://localhost:3000/(https://www.gravatar.com/avatar/00000000000000000000000000000000?d=mp&f=y)) on `next/image`, hostname "www.gravatar.com" is not configured under images in your `next.config.js`
```

right now our default image is still hosted in gravatar.com , I just leave it as it is for now. 
In the future we might 
- use something like this https://ui-avatars.com/
- or host the profile picture in our own CDN

yeah anyway, right now the image still very light. It's okay without next image  for now. We can optimize later.